### ISSUE TITLE
Can't get TypeScript problem squigglies to go away after adding new NPM dependency (need to close/re-open folder or restart VS Code)

### ISSUE THREAD

**Issue by jaredkrinke** [microsoft/TypeScript#36365]
(Not a blocking problem, but annoying)

Is it possible to re-run code analysis after adding a new NPM dependency? I couldn't find a way (other than restarting or closing+reopening the folder), and didn't see any existing issues mentioning this scenario.

- VSCode Version: 1.41.1
- OS Version: Windows 10.0.18362

Steps to Reproduce:

1. Write TypeScript code that imports a package you haven't added as a dependency yet (VS Code will correctly flag the import as an error, indicating it can't find the module), e.g. import * as MyLib from "my-lib";
2. In a terminal, run e.g. "yarn add my-lib" to add the missing dependency (note: at this point, "yarn build" succeeds, so the code is now valid)

Expected:
Either VS Code notices that package.json (or node_modules) has changed and re-runs TypeScript code analysis (resolving the problem squiggly from step 1) OR there is some way to trigger re-running of code analysis (also resulting in the problem going away, since the issue was fixed in stpe 2).

Actual:
The problem squiggly from step 1 doesn't go away without drastic measures. Saving/closing/reopening the affected source files doesn't help. Closing the folder and reopening or restarting VS Code work, but these are slow and clunky.

**Comment by mjbvz** [comment]
Having trouble reproducing this on Mac OS at least

@jaredkrinke Can you please provide specific repo steps (i.e. a small project with a specific package to try installing)

**Comment by jaredkrinke** [comment]
This started working for me, even though my VS Code and OS versions don't appear to have changed. Closing...

Version: 1.41.1 (system setup)
Commit: 26076a4de974ead31f97692a0d32f90d735645c0
Date: 2019-12-18T14:58:56.166Z
Electron: 6.1.5
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: Windows_NT x64 10.0.18362

**Comment by jaredkrinke** [comment]
Note: it's possible I misinterpreted this originally. If I'm working on in a Node.js environment and I add something like `import * as assert from "assert";` I get a red squiggly that doesn't go away after `yarn add @types/node` and adding `"esModuleInterop": true` to my `tsconfig.json` file.

So maybe I conflated "adding a new module" with "changing compiler options in tsconfig.json" -- I just happened to do both at the same time.
